bot/seasons/evergreen/snakes/snakes_cog.py

```diff
@@ -1049,16 +1057,16 @@ def predicate(reaction_: Reaction, user_: Member):
 
     @snakes_group.command(name='help')
     async def help_command(self, ctx: Context):
-        """
-        This just invokes the help command on this cog.
-        """
+        """Invokes the help command for the Snakes Cog."""
+
         log.debug(f"{ctx.author} requested info about the snakes cog")
         return await ctx.invoke(self.bot.get_command("help"), "Snakes")
 
     @snakes_group.command(name='snakify')
     async def snakify_command(self, ctx: Context, *, message: str = None):
         """
-        How would I talk if I were a snake?
+        How would I talk if I were a snake.
```

Comment:
This doesn't really read well either. How about something like
```suggestion
        Shows the given message in snake-speak.
```